### ISSUE TITLE
The parameters of LayoutXLM are not properly initialized

### ISSUE THREAD

**Issue by code7thday** [microsoft/unilm#399]
When I am using LayoutXLM pretrained model to initialize a LayoutXLM-base model, It seems that the pretrained model's weights are not loaded correctly.

**Running code:**
from layoutlmft.models.layoutxlm import LayoutXLMModel
from transformers import AutoTokenizer

model_path="microsoft/layoutxlm-base"
tokenizer = AutoTokenizer.from_pretrained(model_path)
model = LayoutXLMModel.from_pretrained(model_path)

**I got:**
Some weights of the model checkpoint at microsoft/layoutxlm-base were not used when initializing LayoutXLMForTokenClassification: ['layoutlmv2.visual.backbone.bottom_up.stem.conv1.norm.num_batches_tracked', 'layoutlmv2.visual.backbone.bottom_up.res2.0.shortcut.norm.num_batches_tracked', 'layoutlmv2.visual.backbone.bottom_up.res2.0.conv1.norm.num_batches_tracked', 'layoutlmv2.visual.backbone.bottom_up.res2.0.conv2.norm.num_batches_tracked', 'layoutlmv2.visual.backbone.bottom_up.res2.0.conv3.norm.num_batches_tracked', 
...
]
- This IS expected if you are initializing LayoutXLMModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing LayoutXLMForTokenClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).

Is it normal that all those weights are not initialized?

**Comment by NielsRogge** [comment]
See #343 (it's normal, these parameters are batch normalization statistics which get a value when you fine-tune the model)

**Comment by ranpox** [comment]
Also, check out https://github.com/facebookresearch/detectron2/issues/1340
